src/main/java/seedu/address/logic/commands/CommandResult.java

```diff
@@ -17,6 +17,9 @@ public class CommandResult {
     /** The application should exit. */
     private final boolean exit;
 
+    /** Flag between Task command [true] and Person command [false] */
+    private boolean isTaskCommand = false;
+
```

Comment:
So far this will cause the `help` command to switch to the Person tab. Would it be better to have a "don't switch" value for this?

Reply:
sth like https://github.com/AY2223S1-CS2103T-T14-2/tp/commit/91a5d8059bf13bbd42783c4f1785a7c560d41275 I guess?

Reply:
That sounds easier to understand!